### PR TITLE
Gpool double acquire fix

### PR DESCRIPTION
**Pull request by emulbreh**

### CODE REVIEW

**Comment by mouadino** [comment]
@emulbreh why change the package name ?

**Comment by emulbreh** [comment]
Because it's not a contrib package. It's used by core.

File: lymph/utils/gpool.py [hunk 1]
```diff
@@ -15,9 +15,7 @@ class NonBlockingPool(Pool):
     When the ``timeout`` is not given or set to None the pool will reject
     immediately without waiting when pool size reach max size.
 
-    In case ``size`` is None this will create an unbound pool, this was
```
Comment (mouadino) [inline lymph/utils/gpool.py]: Why ?

**Comment by mouadino** [comment]
LGTM

File: lymph/utils/gpool.py (hunk 1 above)
Comment (emulbreh) [inline lymph/utils/gpool.py]: Because it's a misleading comment (it's not true at the moment), unbounded pools are the default.

**Pull request merged**